packages/client-adapters/src/adapters/lexicons/adapter.js

```diff
@@ -250,7 +257,12 @@ class AlpheiosLexiconsAdapter extends BaseAdapter {
   */
   async updateFullDefs (fullDefsRequests, config, homonym) {
     for (let request of fullDefsRequests) { // eslint-disable-line prefer-const
-      const fullDefDataRes = this.fetch(request.url, { type: 'xml' })
+      let fullDefDataRes
+      if (cachedDefinitions.has(request.url)) {

```

Comment:
so were we never checking the cached definitions before?

Reply:
Yes, there were not such caching for full definitions before.
It would be useful both for definitions and for fixtures.